spring-cloud-stream-binder-kinesis-docs/src/main/asciidoc/overview.adoc

```diff
@@ -458,15 +458,15 @@ awslocal kinesis create-stream --stream-name my-test-stream --shard-count 1
 
 awslocal dynamodb create-table \
 --table-name spring-stream-lock-registry \
---attribute-definitions AttributeName=lockKey,AttributeType=S AttributeName=sortKey,AttributeType=S \
---key-schema AttributeName=lockKey,KeyType=HASH AttributeName=sortKey,KeyType=RANGE \
+--attribute-definitions AttributeName=lockKey,AttributeType=S \
+--key-schema AttributeName=lockKey,KeyType=HASH \
 --provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5 \
 --tags Key=Owner,Value=localstack
 
 awslocal dynamodb create-table \
 --table-name spring-stream-metadata \
---attribute-definitions AttributeName=KEY,AttributeType=S \
---key-schema AttributeName=KEY,KeyType=HASH \
+--attribute-definitions AttributeName=metadataKey,AttributeType=S \
+--key-schema AttributeName=metadataKey,KeyType=HASH \
```

Comment:
And yes: starting with version `3.0` the `spring-integration-aws` has this key renamed since `KEY` is really a key word and `metadataKey` is more cleaner name anyway.